src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -116,8 +116,9 @@
                     ? tooltipNode.closest(tooltipNode.dataset.tooltipContainerSelector)
                     : 'body';
                 const iframe = document.querySelector(tooltipNode.dataset.tooltipIframeSelector);
+                const tooltipInstance = bootstrap.Tooltip.getInstance(tooltipNode);
 
-                tooltipNode.dataset.originalTitle = tooltipNode.title;
+                tooltipNode.dataset.originalTitle = tooltipInstance ? tooltipInstance._getTitle() : tooltipNode.title;
 
                 new bootstrap.Tooltip(tooltipNode, {
```

Comment:
We are creating a tooltip instance even in a case in which `tooltipInstance` returned by `getInstance` is not `null`? Isn't this duplication of instances? 🤔